website/public/js/controllers/settingsCtrl.js

```diff
@@ -66,12 +66,23 @@ habitrpg.controller('SettingsCtrl',
       User.set({'flags.newStuff':true});
     }
 
-    $scope.saveDayStart = function(){
-      var dayStart = +User.user.preferences.dayStart;
-      if (_.isNaN(dayStart) || dayStart < 0 || dayStart > 24) {
+    $scope.passDayStart = User.user.preferences.dayStart;
```

Comment:
I get what it's doing, and there's no problem with assigning a different variable name for the saveDayStart argument, it's just not intuitive to me what the name `passDayStart` means. Your explanation makes sense to me, but I think we can come up with a clearer name for it.